README.md

```diff
@@ -294,6 +295,33 @@ Following the definition of `NaN` from the IEEE:
 >
 > &mdash; [“What is the rationale for all comparisons returning false for IEEE754 NaN values?”](https://stackoverflow.com/questions/1565164/1573715#1573715) at StackOverflow
 
+## Object.<i></i>is() and === weird cases
```

Comment:
What `<i></i>` stands for?

Reply:
I guess I was too caught up by the content, I forgot to remove the i tags. Thanks for pointing this out too 
(Btw, that is a hack I'm using in my notes that are on a different website to avoid turning Object.is into a link. I copy and pasted from my notes, hence why I forgot to remove it)

Reply:
Please remove it